### ISSUE TITLE
Add a config option to murmur.ini to set superuser password

### ISSUE THREAD

**Issue by C0rn3j** [mumble-voip/mumble#3226]
I compiled 1.3.x from git - wanting to set a superuser password requires to annoyingly launch murmur manually with a parameter.

I'd welcome an option to set superuser option via a config line in murmur.ini.

https://wiki.mumble.info/wiki/Murmurguide#Linux_4

If there are other ways to set the superuser password then they are not noted in the wiki.

EDIT: Looks like the superuser password is autogenerated at first start of the server and is in logs. Would still welcome a config option tho.

**Comment by mkrautz** [comment]
Typically, config options in murmur.ini are the 'defaults' which are then inherited by virtual servers unless a per-vserver option is set.

That makes a murmur.ini option for the SuperUser a little weird: you probably do not want the same SuperUser password on all virtual servers.

Then there's the issue of storing the password in plain text. I'm not going to argue very strongly on that front, because the fact is that we already emit the auto-generated password to the log file in plain text.

One could imagine that a super user password set in murmur.ini would be global, and allow you to log into all virtual servers as SuperUser. But each virtual server would still have its own local password, too.
So a global SuperUser password that you can use instead (or alongside) the per-vserver's SuperUser password.

**Comment by C0rn3j** [comment]
>One could imagine that a super user password set in murmur.ini would be global, and allow you to log into all virtual servers as SuperUser. But each virtual server would still have its own local password, too.

That sounds great. I don't use the virtual servers feature so I didn't even think of that as an issue.

**Comment by DesolatePsyche** [comment]
I know this is old thread... but whenever I install mumble server, log simply doesn't show on my NUC/server PC... when I tried to install it on my main PC, log/console showed password. And I simply cannot figure out how fix it...

**Comment by Krzmbrzl** [comment]
If you start up a new server instance (that has not yet have an associated database entry), a new password will be generated and logged. If you just start an already existing server, no new password will be generated and therefore nothing will be logged.

**Comment by DesolatePsyche** [comment]
> If you start up a new server instance (that has not yet have an associated database entry), a new password will be generated and logged. If you just start an already existing server, no new password will be generated and therefore nothing will be logged.

I did fresh install on it. Tried the launch parameter -supw <password> as well, but that simply instantly closed the server.

**Comment by Krzmbrzl** [comment]
Yes, it sets the password and exits. Afterwards the password is set and you can start the server normally.

**Comment by DesolatePsyche** [comment]
> Yes, it sets the password and exits. Afterwards the password is set and you can start the server normally.

yup, this method worked. I wasn't sure whenever it worked as .exe instantly closed so didn't figure testing with it. Thanks.
